src/lib.rs

```diff
@@ -137,6 +141,16 @@ pub trait GenerateVerifiable {
 		message: &[u8],
 	) -> Result<(Self::Proof, Alias), ()>;
 
+	/// Generate a batch verification step out of a proof and the information needed to verify it
+	/// independently. The result of this operation should be used in a batch verification operation
+	/// along with other such steps.
+	fn batch_step(
```

Comment:
I don't understand what is the usage of this method. E.g. I see that in the ring implementation it just groups the information in a tuple